pkg/metrics/metrics.go

```diff
@@ -166,12 +166,9 @@ The label 'reason' can have the following values:
 - "ClusterQueueStopped" means that the workload was evicted because the ClusterQueue is stopped.
 - "LocalQueueStopped" means that the workload was evicted because the LocalQueue is stopped.
 - "NodeFailures" means that the workload was evicted due to node failures when using TopologyAwareScheduling.
-- "Deactivated" means that the workload was evicted because spec.active is set to false.
-- "DeactivatedDueToAdmissionCheck" means that the workload was evicted and deactivated by Kueue due to a rejected admission check.
-- "DeactivatedDueToMaximumExecutionTimeExceeded" means that the workload was evicted and deactivated by Kueue due to maximum execution time exceeded.
-- "DeactivatedDueToRequeuingLimitExceeded" means that the workload was evicted and deactivated by Kueue due to requeuing limit exceeded.`,
+- "Deactivated" means that the workload was evicted because spec.active is set to false.`,
```

Comment:
@mykysha Could you list the possible values for underlying_cause like the following?

```suggestion
- "Deactivated" means that the workload was evicted because spec.active is set to false.
The label 'underlying_cause' can have the following values:
- "" means that the value in 'reason' label is the root cause for eviction,
- "AdmissionCheck" means that the AdmissionCheck state is the behind cause of eviction
AND MORE...`,
```

Same comment on ohter metrics as well.

Reply:
Added label descriptions